README.md

```diff
@@ -186,6 +186,23 @@ $pricing->amount; // example: 11.5
 $pricing->currency; // example: "EUR"
 ```
 
+### Get a job ETA
+
+Before creating a Job you can ask for ETA. Asking for ETA is **optional** and does not prevent you from creating a job.
```

Comment:
I would write instead:

"Before creating a Job you can ask for an estimated time of arrival at the pickup location (expressed in seconds). Asking for ETA is **optional** and does not prevent you from creating a job."

WDYT?

Reply:
Consider it done